css/index.css

```diff
@@ -1,3 +1,461 @@
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/*************************************************
+******                                       *****
+******    These Styles are for every page    *****
+******                                       *****
+**************************************************/
+
+
+.test {
+	border: 2px solid red;
+}
+
+.container {
+	max-width: 1000px;
+	margin: 0 auto;
+	padding: 15px 0;
+	padding-top: 130px;
+}
+
+@media (max-width: 800px) {
+    .container {
+        width: 94%;
+        margin: 0 3%;
+    }
+}
+
+@media (max-width: 500px){
+    .container {
+        margin-top: 270px;
+    }
+}
+
+.banner {
+    position: fixed;
+    width: 100%;
+	top: 0;
+	box-shadow: 2px 0 3px black;
+    background: rgba(255,255,255, 0.7);
+}
+
+@media (max-width: 800px) {
+    .banner {
+        position: absolute;
+        width: 100%;
+        box-shadow: none;
+    }
+}
+
+/* Styling our banner */
+.banner-wrap {
+    display: flex;
+	justify-content: space-between;
+    align-items: center;
+    
+    max-width: 1000px;
+    margin: 0 auto;
+}
+
+@media (max-width: 800px) {
+    .banner-wrap {
+        flex-direction: column-reverse;
+        padding-top: 30px;
+        width: 94%;
+        margin: 0 3%;
+    }
+}
+
+.banner .banner-wrap nav {
+	max-width: 1000px;
+    padding: 15px 0;
+    width: 70%;
+    display: flex;
+    justify-content: space-between;
+}
+
+@media (max-width: 800px) {
+    .banner .banner-wrap nav {
+        width: 95%;
+        margin: 22px auto 0;
+        font-size: 14px;
+    }
+}
+
+@Media (max-width: 500px){
+    .banner .banner-wrap nav{
+        flex-direction: column;
+        align-items: center;
+        text-align: center;
+        padding: 0;
+        margin: 30px 0 0;
+    }
+}
+
+.banner .banner-wrap nav a {
+	display: inline-block;
+	font-family: inherit;
+	color: lightslategray;
+	text-decoration: none;
+
+	margin-left: 30px;
+}
+
+@Media (max-width: 800px){
+    .banner .banner-wrap nav a {
+        margin: 0;
+        padding: 15px 0;
+    }
+}
+
+@Media (max-width: 500px){
+    .banner .banner-wrap nav a {
+        width: 100%;
+        font-size: 24px;
+        padding: 0 auto;
+        
+        border-top: 1px solid black;
```

Comment:
I'd get rid of this border-top. now the lines will be nice and uniform. only problem left is the hover looks heckin weird. if you wanna use scale i think the only way to cleanly do it might be to wrap each <a> in a div and apply the borders to the divs and scale to the a. before, the whole a tag was scaling up which made its overlapping borders pop out and get a lil crazy. this is all just design preference though, might be something to play with!